api/commands/compile.ts

```diff
@@ -50,11 +49,7 @@ export async function compile(
   const encodedGraphInBase64 = await CompileChildProcess.forkProcess().compile(compileConfig);
   const encodedGraphBytes = new Uint8Array(util.base64.length(encodedGraphInBase64));
   util.base64.decode(encodedGraphInBase64, encodedGraphBytes, 0);
-  const compiledGraph = dataform.CompiledGraph.decode(encodedGraphBytes);
-  return dataform.CompiledGraph.create({
-    ...compiledGraph,
-    graphErrors: validate(compiledGraph)
-  });
+  return dataform.CompiledGraph.decode(encodedGraphBytes);
```

Comment:
my understanding is that you kept the UI code backwards-compatible with validation errors, is that right? if so, removing this part will mean that the backwards-compatible code never gets called. (that's fine with me, but an FYI to you)

Reply:
Ah yup, in that change I added a small bit of code where the compiled graph is retrieved that converts validation errors to compilation errors. I'll remove that soon after this.